README.md

```diff
@@ -241,8 +241,13 @@ Follow the interactive prompt to generate a post with pre-filled front matter.
 **Vercel**  
 The easiest way to deploy the template is to use the [Vercel Platform](https://vercel.com) from the creators of Next.js. Check out the [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
 
-**Netlify / GitHub Pages / Firebase etc.**  
-As the template uses `next/image` for image optimization, additional configurations have to be made to deploy on other popular static hosting websites like [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/). An alternative image optimization provider such as Imgix, Cloudinary or Akamai has to be used. Alternatively, replace the `next/image` component with a standard `<img>` tag. See [`next/image` documentation](https://nextjs.org/docs/basic-features/image-optimization) for more details.
+**Netlify**
+[Netlify](https://www.netlify.com/)’s Next.js Runtime configures your site on Netlify to enable key Next.js functionality without the need for additional configurations. It automatically generates serverless functions that handle server-side rendered (SSR) pages, incremental static regeneration (ISR), `next/images`, and other Next.js features.
```

Comment:
This is copied verbatim form the Netlify docs. Maybe we should reformulate it to avoid copyright issues.

Reply:
my bad this was done a bit hastily last night. should be ok in the new commit.